dev/com.ibm.ws.microprofile.health.2.0_fat/fat/src/com/ibm/ws/microprofile/health20/fat/DelayAppStartupHealthCheckTest.java

```diff
@@ -194,6 +195,9 @@ public void run() {
                     }
 
                 }
+            } catch (AssumptionViolatedException ave) {
```

Comment:
If you have caught the `AssumptionViolatedException`, will the `while (repeat)` loop continue?

Reply:
Re-factored the code to remove the `Assume.assumeTrue(b)` out of the try/catch block, to avoid this.